libs/sdk/src/eth_rpc.rs

```diff
@@ -0,0 +1,70 @@
+use crate::http::http_post_json;
+use alloy::{hex::ToHexExt, primitives::Bytes};
+use serde::{Deserialize, Serialize};
+
+#[derive(Debug, Clone, Serialize, Deserialize)]
+struct RequestEthCallParams {
+    data: String,
+    from: String,
+    to: String,
+}
+
+#[derive(Debug, Clone, Serialize, Deserialize)]
+struct RequestEthCall {
+    jsonrpc: &'static str,
+    method: &'static str,
```

Comment:
Why `'static` here? Is it out of laziness for specifying and propagating the lifetime or is it actually a meaningful limitation?